Override default settings (#1887)
ezsystems/developer-documentation: docs/customer_management/cp_configuration.md

## Roles and Policies

You can add custom roles to your installation

> by adding them to `ibexa.site_access.config.default.corporate_accounts.roles`

Reviewer: ```suggestion
by adding them under the `ibexa.site_access.config.default.corporate_accounts.roles` key
```

(to the default already set in `vendor/ibexa/corporate-account/src/bundle/Resources/config/default_settings.yaml`):

```yaml

> parameters:

Reviewer: Could we actually have a full example of the override here?

Author: @DominikaK I'm not sure about what you're asking for.
I'll make test at lest to validate that the custom role below is added to the existing ones and replacing the whole.

Reviewer: Sorry, disregard that comment, I misunderstood.

    ibexa.site_access.config.default.corporate_accounts.roles:
        custom_role: Company Assistant
```

> You can do it per Site Access or Site Access grup by using [SiteAccess-aware configuration](multisite/siteaccess_aware_configuration.md).

Reviewer: ```suggestion
You can do it per SiteAccess or SiteAccess group by using [SiteAccess-aware configuration](siteaccess_aware_configuration.md).
```

Author: "grup" is so cute…

> You can do it per Site Access or Site Access grup by using [SiteAccess-aware configuration](multisite/siteaccess_aware_configuration.md).

Reviewer: Good catch!

## Roles and Policies

You can add custom roles to your installation
by listing them under the `ibexa.site_access.config.default.corporate_accounts.roles` key

> (it overwrites the default list set in `vendor/ibexa/corporate-account/src/bundle/Resources/config/default_settings.yaml`, the following example keep them by redeclaring them):

Author: @DominikaK Test done, it doesn't add, it replaces. I'm not satisfied with my wording. Can you help me, please?

Author: Done in 0e63874a3c9db3e891de85e1ec877437d62bef18

> by listing them under the `ibexa.site_access.config.default.corporate_accounts.roles` key
> (it overwrites the default list set in `vendor/ibexa/corporate-account/src/bundle/Resources/config/default_settings.yaml`, the following example keep them by redeclaring them):

Reviewer: ```suggestion
by listing them under the `ibexa.site_access.config.default.corporate_accounts.roles` key.
This key overwrites the default list set in `vendor/ibexa/corporate-account/src/bundle/Resources/config/default_settings.yaml` (the following example redeclares them for clarity):
```
